src/foraging_gui/RigJsonBuilder.py

```diff
@@ -105,7 +108,6 @@ def build_rig_json_core(settings, water_calibration, laser_calibration):
                         sensor_format_unit=SizeUnit.IN,
                         sensor_width=720,
                         sensor_height=540,
-                        max_frame_rate=522,
```

Comment:
Is max_frame_rate not part of the schema anymore?

Reply:
No it's not